src/AppBundle/Model/Event.php

```diff
@@ -617,6 +620,31 @@ public function getWikis(): Collection
         return $this->wikis;
     }
 
+    /**
+     * Get all of this event's wikis that do not yet have at least one category defined.
+     * Wikidata is excluded because it can never have categories.
+     * @return ArrayCollection Collection of EventWiki objects.
+     */
+    public function getWikisWithoutCategories(): ArrayCollection
+    {
+        if (null !== $this->wikisWithoutCategories) {
+            return $this->wikisWithoutCategories;
+        }
+        $this->wikisWithoutCategories = $this->wikis->filter(function (EventWiki $eventWiki) {
+            // Wikidata never has categories, so we don't return it in this list.
+            if ('www.wikidata' === $eventWiki->getDomain()) {
+                return false;
+            }
+            foreach ($this->categories as $cat) {
+                if ($cat->getDomain() === $eventWiki->getDomain()) {
+                    return false;
+                }
+            }
+            return true;
+        });
+        return $this->wikisWithoutCategories;
+    }
```

Comment:
I think this method deserves a unit test

Reply:
Done.